src/sentry/api/endpoints/group_hashes.py

```diff
@@ -79,14 +79,17 @@ def __handle_results(self, project_id, group_id, user, results):
         return [self.__handle_result(user, project_id, group_id, result) for result in results]
 
     def __handle_result(self, user, project_id, group_id, result):
-        event = {
-            "timestamp": result["latest_event_timestamp"],
-            "event_id": result["event_id"],
-            "group_id": group_id,
-            "project_id": project_id,
-        }
+        event = SnubaEvent(
+            {
+                "timestamp": result["latest_event_timestamp"],
+                "event_id": result["event_id"],
+                "group_id": group_id,
+                "project_id": project_id,
+            }
+        )
+        event.bind_node_data()
```

Comment:
What does this become after SnubaEvent is gone ?

Reply:
This has been fixed in https://github.com/getsentry/sentry/pull/16286, will rebase to pull in those changes